python/sglang/srt/configs/model_config.py

```diff
@@ -287,7 +295,7 @@ def get_total_num_kv_heads(self) -> int:
             num_kv_heads = getattr(self.hf_text_config, attr, None)
             if num_kv_heads is not None:
                 return num_kv_heads
-
+            
```

Comment:
The lint seems failing, you can run `pre-commit run --all-files` to fix it if the PR is ready for review. Otherwise feel free to ignore me.